packages/devextreme-schematics/src/install/index.ts

```diff
@@ -12,22 +12,20 @@ import {
   addPackageJsonDependency,
 } from '@schematics/angular/utility/dependencies';
 
-import {
-  NodePackageInstallTask
-} from '@angular-devkit/schematics/tasks';
-
 import { latestVersions } from '../utility/latest-versions';
 import { modifyJSONFile } from '../utility/modify-json-file';
 import { getProjectName } from '../utility/project';
 
+import * as patch from '../utility/patch';
```

Comment:
You can use named import: import { CustomNodePackageInstallTask } from '../utility/patch'

Reply:
https://github.com/DevExpress/devextreme-cli/pull/793/commits/0527ba123b3e8516dfcfe824a318010c487fb3a5